### PR TITLE
Adding expense limits to user screen

### PR DESCRIPTION
**Pull request by ricardofreitasrocha**
### Change description ###

!image-20240716-155339.png|width=846,height=641,alt="image-20240716-155339.png"!

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by Ben-Edwards-cgi — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/juror/api/moj/service/UserServiceModImpl.java [hunk 1]
```diff
@@ -257,4 +259,10 @@ String createUsername(String email) {
     boolean doesUserExistWithEmail(String email) {
         return userRepository.existsByEmail(email.trim());
     }
+
+    private void updateUserApprovalLimit(User user, UpdateUserDto updateUserDto) {
+        if (SecurityUtil.isAdministration() || SecurityUtil.isManager() && SecurityUtil.isCourt()) {
```
Comment (Ben-Edwards-cgi) [inline src/main/java/uk/gov/hmcts/juror/api/moj/service/UserServiceModImpl.java:264]: Would be good to have this as its own method like a private boolean canEditApprovalLimit()

File: src/main/java/uk/gov/hmcts/juror/api/moj/domain/authentication/UserDetailsDto.java [hunk 2]
```diff
@@ -49,5 +53,9 @@ public UserDetailsDto(User user, List<UserCourtDto> courts) {
         this.userType = user.getUserType();
         this.roles = user.getRoles();
         this.courts = courts;
+
+        if (SecurityUtil.isAdministration() || SecurityUtil.isManager() && SecurityUtil.isCourt()) {
```
Comment (Ben-Edwards-cgi) [inline src/main/java/uk/gov/hmcts/juror/api/moj/domain/authentication/UserDetailsDto.java:57]: Would be good to have this as its own method like a private boolean canEditApprovalLimit()

Comment (ricardofreitasrocha) [reply src/main/java/uk/gov/hmcts/juror/api/moj/domain/authentication/UserDetailsDto.java]: updated

File: src/main/java/uk/gov/hmcts/juror/api/moj/service/UserServiceModImpl.java (hunk 1 above)
Comment (ricardofreitasrocha) [reply src/main/java/uk/gov/hmcts/juror/api/moj/service/UserServiceModImpl.java]: updated

**Pull request merged**